javascript/ql/test/query-tests/Security/CWE-022/TaintedPath/handlebars.js

```diff
@@ -34,7 +34,7 @@ app.get('/some/path2', function (req, res) {
 });
 
 app.get('/some/path3', function (req, res) {
-    res.send(data.compiledUnknown({ name: req.params.name })); // NOT ALLOWED (could be using a vulnerable helper)
+    res.send(data.compiledUnknown({ name: req.params.name })); // ALLOWED (could be using a vulnerable helper, but we'll assume it's ok)
```

Comment:
We tend to use `// NOT OK` / `// OK`, or sometimes `// GOOD` / `// BAD`.  
(That also works with the consistency checking when we use that).

Reply:
The alert location is in a different line (11), adding `NOT OK` here will break the existing consistency test. This comment is really only intended for a human